### PR TITLE
Create com.kavilgroup.gelectrical.json

### PR DESCRIPTION
**Pull request by manuvarkey**
### Please confirm your submission meets all the criteria

- [x ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### Remarks
`--filesystem=host` permission has been added so that the application during report generation phase can create the required files within the user selected directory.

### CODE REVIEW

**Comment by manuvarkey** [comment]
Project homepage

[https://github.com/manuvarkey/GElectrical](https://github.com/manuvarkey/GElectrical)

File: com.kavilgroup.gelectrical.json [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+                    "only-arches": [
+                        "x86_64"
+                    ]
+                }
+            ]
+        },
+        {
+            "name" : "gelectrical",
+            "buildsystem" : "simple",
+            "cleanup" : [
```
Comment (hfiguiere) [inline com.kavilgroup.gelectrical.json:412]: cleanup doesn't do what you think it does.

It's not to clean the build dir.

File: com.kavilgroup.gelectrical.json [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+    "app-id" : "com.kavilgroup.gelectrical",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gelectrical_launcher.py",
+    "finish-args": [
+           "--socket=wayland",
+           "--socket=fallback-x11",
+           "--share=ipc",
+           "--filesystem=host"
```
Comment (hfiguiere) [inline com.kavilgroup.gelectrical.json:11]: that's a very broad filesystem permission

**Comment by hfiguiere** [comment]
bot, build com.kavilgroup.gelectrical

File: com.kavilgroup.gelectrical.json (hunk 2 above)
Comment (manuvarkey) [reply com.kavilgroup.gelectrical.json:11]: Actually I needed something like this

https://github.com/flatpak/xdg-desktop-portal/issues/463

Currently this permission seems to be the only option.

File: com.kavilgroup.gelectrical.json (hunk 1 above)
Comment (manuvarkey) [reply com.kavilgroup.gelectrical.json:412]: My bad! Will change that.

**Comment by manuvarkey** [comment]
Test build is working fine.

File: com.kavilgroup.gelectrical.json (hunk 1 above)
Comment (manuvarkey) [reply com.kavilgroup.gelectrical.json]: Removed with latest commit.

**Pull request closed without merging**